src/ploomber/telemetry/telemetry.py

```diff
@@ -107,39 +114,41 @@ def get_env():
         return 'local'
 
 
-# Checks if a specific directory/file exists, creates if not
-# In case the user didn't set a custom dir, will turn to the default home
-def check_dir_file_exist(name=None, is_file=None):
+def check_dir_exist(name=None):
+    """
+    Checks if a specific directory exists, creates if not.
+    In case the user didn't set a custom dir, will turn to the default home
+    """
     if PLOOMBER_HOME_DIR:
-        final_location = os.path.expanduser(PLOOMBER_HOME_DIR)
+        final_location = PLOOMBER_HOME_DIR
     else:
-        final_location = os.path.expanduser(DEFAULT_HOME_DIR)
+        final_location = DEFAULT_HOME_DIR
 
     if name:
-        final_location = os.path.join(final_location, name)
+        final_location = f"{final_location}/{name}"
```

Comment:
better to use `Path(final_location, name)` instead of `/` - since it may break on windows